coral/coral/media/js/views/components/plugins/licensing-workflow.js

```diff
@@ -546,16 +557,6 @@ define([
                     resourceModelDigitalObjectNodeGroupId: '316c7d1e-8554-11ea-aed7-f875a44e0e11',
                     fileObjectNamePrefix: 'Site report files for '
                   }
-                },
```

Comment:
Currently getting a cardinality error for the digital file here. This might not be due to this PR though as it's an ongoing issue with the edit workflow